src/main/scala/edu/ie3/osmogrid/io/input/InputDataProvider.scala

```diff
@@ -92,8 +99,8 @@ object InputDataProvider extends ActorStopSupport[ProviderData] {
                 OsmReadFailed(exception)
             }
             readOsmData(providerData, replyTo)
```

Comment:
This is a point very loosely adjacent to this PR, but could you please rename the method `readOsmData`, as its functionality does have nothing to do with its current name?

Reply:
Isn't that quite close to what it does? The `InputDataProvider` upon requests reads the osm data with its `OsmSource` by piping to self so it actually resides in `readOsmData` during that time and handles the results of the read process accordingly.